### DISCUSSION TITLE
<title>Github pages problem

### DISCUSSION THREAD

**Discussion by subirvarma** [community/community#57649] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,
I am using Mathjax to format math equations on github pages. It worked for for the past year or so, but I have noticed recently that whenever I put in a math equation into the .md document, the system stops including figures from that point onwards. Has anyone else noticed this problem?

Thanks,
Subir

**Comment by DSDmark** [comment]
Hi @subirvarma,

I think there was conflict between the way that MathJax and GitHub pages handle images.

it may help you:- [**issue**](https://github.com/plotly/plotly.py/issues/360)
